app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileResetPinFragmentTest.kt

```diff
@@ -1006,6 +1009,38 @@ class ProfileResetPinFragmentTest {
     }
   }
 
+  @Test
+  fun testFragment_fragmentLoaded_verifyCorrectArgumentsPassed() {
+    ActivityScenario.launch<ProfileResetPinActivity>(
+      ProfileResetPinActivity.createProfileResetPinActivity(
+        context = context,
+        profileId = 0,
+        isAdmin = true
```

Comment:
Is there an expectation that the app behaviour will be different if `isAdmin = false`? If so, can we add another test for that too?

Reply:
I'll review it and let you know.

Reply:
Yes, There is little change in app behavior.
If isAdmin = false, it asks for 3 digit pin.
If isAdmin = true, it asks for 5 digit pin. 

I tested the argument with isAdmin = false, it passed Expresso and the robolectric test.
Should I add this test?

Reply:
Ok.

Reply:
Is this test name appropriate for the test? Fragment_fragmentLoaded_whenIsAdminFalse_verifyCorrectArgumentsPassed